### PR TITLE
fix Slavic support

### PR DESCRIPTION
**Pull request by feelingit**
Fixing rules for: 

5-20,
**X**05-**X**20, 
**XX**05-**XX**20, 
**XXX**05-**XXX**20, 
... etc.

### CODE REVIEW

File: humanize-duration.js [hunk 1]
```diff
@@ -496,7 +496,7 @@
   function getSlavicForm (c) {
     if (Math.floor(c) !== c) {
       return 2
-    } else if ((c >= 5 && c <= 20) || (c % 10 >= 5 && c % 10 <= 9) || c % 10 === 0) {
+    } else if ((c%100 >= 5 && c%100 <= 20) || (c % 10 >= 5 && c % 10 <= 9) || c % 10 === 0) {
```
Comment (EvanHahn) [inline humanize-duration.js]: It looks like there was a linting error here. I think this needs to be:

```js
} else if ((c % 100 >= 5 && c % 100 <= 20) || (c % 10 >= 5 && c % 10 <= 9) || c % 10 === 0) {
```

Comment (feelingit) [inline humanize-duration.js]: Sorry, fixed.

**Comment by EvanHahn** [comment]
This looks great! Could you add new tests to `test/definitions/ru.csv` and `test/definitions/uk.csv` to make sure this doesn't happen again?

**Comment by feelingit** [comment]
I added some examples to `test/definitions/ru.csv` and `test/definitions/uk.csv`. I'm not sure about uk, but i think it's the same.

**Comment by EvanHahn** [comment]
This has been fixed in version 3.9.1. Thank you!

**Pull request merged**